.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
 install: pip install pytest "jupyterlab~=1.0"
 script:
   - python setup.py sdist
-  - pip install jupyterlab_git[test] --find-links=dist --no-deps --no-cache-dir -v
-  - pip install jupyterlab_git[test] --find-links=dist -v
+  - pip install jupyterlab_git[test] --pre --no-index --find-links=dist --no-deps --no-cache-dir -v
+  - pip install jupyterlab_git[test]
```

Comment:
I suppose it does make sense to remove the `--find-links` flag here. Hopefully we won't ever try to bundle our Python package's Python deps :)